service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java

```diff
@@ -121,13 +122,29 @@ public UsersRecord createNewUser(
     newUser.setPrivilegeLevel(PrivilegeLevel.STANDARD);
     newUser.store();
 
+    AuditRecord audit = db.newRecord(AUDIT);
+    audit.setTableName("users");
+    audit.setTransactionType("insert");
+    audit.setResult(newUser.getId().toString());
+    audit.setUserId(newUser.getId());
+    audit.setTimestamp(new Timestamp(System.currentTimeMillis()));
+    audit.insert();
+
     return newUser;
   }
 
   /** Given a JWT signature, store it in the BLACKLISTED_REFRESHES table. */
   public void addToBlackList(String signature) {
     Timestamp expirationTimestamp = Timestamp.from(Instant.now().plusMillis(msRefreshExpiration));
     db.newRecord(Tables.BLACKLISTED_REFRESHES).values(signature, expirationTimestamp).store();
+
+    AuditRecord audit = db.newRecord(AUDIT);
+    audit.setTableName("blacklisted_refreshes");
+    audit.setTransactionType("insert");
+    audit.setResult(signature);
+    audit.setUserId(-1);
+    audit.setTimestamp(new Timestamp(System.currentTimeMillis()));
+    audit.insert();
```

Comment:
This table likely doesn't need audits. It's only used for authentication purposes.

Reply:
gotcha, will remove